app/controllers/manager/books_controller.rb

```diff
@@ -36,8 +41,12 @@ def update
     end
   
     def destroy
-      @book.destroy
-      redirect_to manager_books_url, notice: t("messages.success.destroy", model: @book.class.name)
+      begin
+        @book.destroy
+        redirect_to manager_books_url, notice: t("messages.success.destroy", model: @book.class.name)
+      rescue 
```

Comment:
what exception are you catching here.
please use Rails Logger to log exceptions as well